lib/gears/timer.lua

```diff
@@ -185,7 +185,6 @@ end
 -- @tparam number timeout Timeout in seconds (e.g. 1.5).
 -- @tparam function callback Function to run.
 -- @treturn timer The timer object that was set up.
--- @see timer.weak_start_new
```

Comment:
"Should work" ? Perhaps.But it doesn't:

/usr/src/awesome-build/lib/gears/timer.lua:212: gears.timer:weak_start_new: module not found: timer